simulation/modelica/initialization/discreteTest06.mos

```diff
@@ -17,16 +17,18 @@ package initializationTests
   initial equation
     pre(y) = 0;
 
-  equation 
+  equation
     der(x) = y;
 
+    // Disallowed in MLS 3.2r1
```

Comment:
You are right. I guess the test should be changed to `when {initial(), time > 0.5} then`.